Cubical/Foundations/Equiv/HalfAdjoint.agda

```diff
@@ -143,3 +163,45 @@ congEquiv {A = A} {B} {x} {y} e = isoToEquiv (iso intro elim intro-elim elim-int
                      ; (j = i0) → secEq e x (i ∨ k)
                      ; (j = i1) → secEq e y (i ∨ k) })
             (secEq e (p j) i)
+
+
+coherent : ∀ {ℓ} {A B : Set ℓ} (isom : Iso A B) → Set ℓ
+coherent (iso f g H K) = ∀ x → cong f (K x) ≡ H (f x)
+
+vogt : ∀ {ℓ} {X Y : Set ℓ} → (isom : Iso X Y) → Σ ((y : Y) → Iso.fun isom (Iso.inv isom y) ≡ y) λ iso' → coherent (iso (Iso.fun isom) (Iso.inv isom) iso' (Iso.leftInv isom))
```

Comment:
vogt?

Reply:
Vogt's lemma, I took inspiration from the proof at:
https://github.com/HoTT/M-types/blob/beebe176981953ab48f37de5eb74557cfc5402f4/function/isomorphism/coherent.agda

Reply:
Cool! Can you please add a reference for this name